src/Evolution/DiscontinuousGalerkin/DgElementArray.hpp

```diff
@@ -4,25 +4,28 @@
 #pragma once
```

Comment:
[Random line] The documentation needs to indicate that this was developed on a local-time-stepping evolution.

If we have data on it, determining whether this works well for global time-stepping (and subcell) would be good, but that's a lot of extra work and is certainly not required for this PR.